### PR TITLE
Creation of Observability Maturity landing page. 

### PR DESCRIPTION
**Pull request by ricegi**
### Give us some context

This PR creates an Observability Maturity landing page with links to existing solution content that federates into the OMA model and placeholders for content that will be going live in the next 30 days. 

### Are you making a change to site code?

NO. No changes to code.

### CODE REVIEW

**Review by a-james-faria — APPROVED** [review]
make it so

File: src/content/docs/new-relic-solutions/observability-maturity/index.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - Implementation guide
+metaDescription: Observability Maturity landing page
+redirects:
+  - /docs/solutions-best-practices-landing-page
+---
+
+<LandingPageHero>
+
+ <HeroContent>
+   New Relic Observability Maturity Architecture (OMA) is now open-source. Observability Maturity Architecture is the practice of discovering, curating, and documenting scalable and repeatable best practices in the Observability Industry. This move will allow all to contribute to the standards of Observability Maturity Architecture.
```
Comment (mmfred) [inline src/content/docs/new-relic-solutions/observability-maturity/index.mdx:19]: Some edit suggestions: 

- remove "now" from the first sentence so it's got longevity.
- lower case "observability industry"

File: src/content/docs/new-relic-solutions/observability-maturity/index.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+   Observability Maturity Architecture is structured by the four primary Observability Value Drivers and their respective use cases, listed below. Use cases are cateloged into templatized "Concepts", "Use Case Implimentation Guides", and "Lesson Plans".
```
Comment (mmfred) [inline src/content/docs/new-relic-solutions/observability-maturity/index.mdx:21]: For a friendlier tone and active voice, you could say, "We've cataloged use cases into concepts..."  Also, I don't know what you mean by "templatized" in this context.

**Review by mmfred — APPROVED** [review]

**Comment by mmfred** [comment]
@ricegi - I provided some feedback, but otherwise this is good to go. The suggestions aren't blocking, and can be addressed any time.

**Pull request merged**